tracer/test/Datadog.Trace.Tests/TraceContextTests.cs

```diff
@@ -234,5 +306,19 @@ public void PartialFlushShouldPropagateMetadata(bool inAASContext)
                 spans.Value.Array[0].GetTag(Tags.AzureAppServicesResourceGroup).Should().BeNull();
             }
         }
+
+        private void SetAASContext(bool inAASContext)
+        {
+            if (!inAASContext)
+            {
+                return;
+            }
```

Comment:
I think you should explicitly set `AzureAppServices.Metadata` here, to make sure there's no issues with ambient context messing with stuff (e.g. I have, in the past, set the AAS vars locally for testing things)

Reply:
Done in 787833a7418157b811f8aeeb747f21bed45d083a